mrtt-ui/src/views/Auth/SignupForm.js

```diff
@@ -0,0 +1,102 @@
+import { Controller, useForm } from 'react-hook-form'
+import { toast } from 'react-toastify'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+import axios from 'axios'
+
+import { ButtonCancel, ButtonSubmit } from '../../styles/buttons'
+import { ButtonContainer, RowFlexEnd } from '../../styles/containers'
+import { ErrorText } from '../../styles/typography'
+import { Form, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import { FormLabel, TextField } from '@mui/material'
+import language from '../../language'
+import LoadingIndicator from '../../components/LoadingIndicator'
+
+const validationSchema = yup.object({
+  email: yup.string().required('Email required'),
+  password: yup.string().required('Password required')
+})
+
+const formDefaultValues = { email: '', password: '' }
+
+const SignupForm = () => {
+  const [isLoading] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const navigate = useNavigate()
+
+  const getBaseUrl = (url) => {
+    const pathArray = url.split('/')
+    const protocol = pathArray[0]
+    const host = pathArray[2]
+    return protocol + '//' + host
+  }
+
+  const authUrl = `${getBaseUrl(process.env.REACT_APP_API_URL)}/users`
```

Comment:
why not just `${process.env.REACT_APP_API_URL}/users`. We can manage if its http or https with .env. Or maybe be more explicit with the get base url function `getProtocol()` 

```
const getProtocol() => {
return  process.env.NODE_ENV === 'production' ? 'https' : 'http'
}
```

(Im making some assumptions of course)

Or maybe just rename the getBaseUrl function to get protocol, and just return the protocol` and let the `authUrl` stitch it together since its already stitching.

Reply:
The env value of `REACT_APP_API_URL` ends in `api/v2`. I just needed the base URL so basically what I need is `...herokuapp.com/users` and not `...herokuapp.com/api/v2/users`.